lib/module.js

```diff
@@ -205,7 +205,7 @@ Module._resolveLookupPaths = function(request, parent) {
   }
 
   var start = request.substring(0, 2);
-  if (start !== './' && start !== '..') {
+  if (request !== '.' && start !== './' && start !== '..') {
```

Comment:
I don't see why this change is necessary. It could also have been written like this to eliminate the `substring` call:

```js
if (request[0] !== '.' || (request[1] !== '/' && request[1] !== '.'))
```

But this isn't a hot path, so it does probably not matter.